libs/core/langchain_core/callbacks/manager.py

```diff
@@ -458,7 +458,7 @@ class RunManager(BaseRunManager):
 
     def on_text(
         self,
-        text: str,
+        text: Union[str, Any],
```

Comment:
> Ah can we just cast the output of SQLDatabase wherever it's passed to on_text instead of changing the on_text method (since it doesn't actually support non-strings)?

Undid this adjustment with bd7a515963.